tony-history-server/buildAndDeploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+DEPLOY_LOG=deploy.log
+
+echo -n "Cleaning up old builds..." > $DEPLOY_LOG
+echo -n "Cleaning up old builds..." 
```

Comment:
To avoid duplicating all these `echo` lines, you could use `tee` to write to stdout and to a file. Example:

```
echo -n "Cleaning up old builds..." | tee $DEPLOY_LOG
```